src/flux/AppDispatcher.js

```diff
@@ -0,0 +1,14 @@
+import { Dispatcher } from 'flux';
```

Comment:
in my opinion flux should be implemented without any libs

Reply:
Hi Yury, Prototype - it is App.js file (NewsApp function). Decorator it is NewsItem.js file -> ItemDecorator function